internal/go_repository.bzl

```diff
@@ -220,6 +220,10 @@ def _go_repository_impl(ctx):
         "http_proxy",
         "https_proxy",
         "no_proxy",
+        "GIT_CONFIG",
```

Comment:
Could you add a `# keep sorted` comment as the first entry of this list? That should have buildifier sort the chunks.

Reply:
@fmeum do you mean the entire list, or the GIT_ part of it?

Reply:
The entire list. Buildifier should then sort the entries in each chunk delimited by comments. Makes it easier for us to recognize duplicates. Please also run buildifier on the file to check that it works.

Reply:
Done